src/components/offer/offer.jsx

```diff
@@ -0,0 +1,348 @@
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+
+      <main className="page__main page__main--property">
+        <section className="property">
+          <div className="property__gallery-container container">
+            <div className="property__gallery">
+
+              {
+                gallery.map((photoSrc, index) => {
+                  return (
+                    <div className="property__image-wrapper" key={index}>
+                      <img className="property__image" src={photoSrc} alt="Photo studio" />
+                    </div>
+                  );
+                })
+              }
+
+            </div>
+          </div>
+          <div className="property__container container">
+            <div className="property__wrapper">
+
+              {premium && (
+                <div className="property__mark">
+                  <span>Premium</span>
+                </div>)
+              }
+
+              <div className="property__name-wrapper">
+                <h1 className="property__name">
+                  {name}
+                </h1>
+                <button className="property__bookmark-button button" type="button">
+                  <svg className="property__bookmark-icon" width="31" height="33">
+                    <use xlinkHref="#icon-bookmark"></use>
+                  </svg>
+                  <span className="visually-hidden">To bookmarks</span>
+                </button>
+              </div>
+              <div className="property__rating rating">
+                <div className="property__stars rating__stars">
+                  <span style={{width: rating}}></span>
+                  <span className="visually-hidden">Rating</span>
+                </div>
+                <span className="property__rating-value rating__value">4.8</span>
+              </div>
+              <ul className="property__features">
+                <li className="property__feature property__feature--entire">
+                  {entire}
+                </li>
+                <li className="property__feature property__feature--bedrooms">
+                  {bedrooms} Bedrooms
+                </li>
+                <li className="property__feature property__feature--adults">
+                  Max {adults} adults
+                </li>
+              </ul>
+              <div className="property__price">
+                <b className="property__price-value">&euro;{price}</b>
+                <span className="property__price-text">&nbsp;night</span>
+              </div>
+              <div className="property__inside">
+                <h2 className="property__inside-title">What&apos;s inside</h2>
+                <ul className="property__inside-list">
+
+                  {
+                    insides.map((insideItem, index) => {
+                      return (
+                        <li className="property__inside-item" key={index}>
```

Comment:
```suggestion
                        <li className="property__inside-item" key={insideItem}>
```
если insideItem уникальные, конечно же